app/controllers/wiki_controller.rb

```diff
@@ -36,6 +36,18 @@ def show
       end
     end
 
+    if @node && @node.has_power_tag('abtest')
+      if current_user == nil || (current_user.role != 'admin' && current_user.role != 'moderator')
+        if Random.rand(2) == 0
+          redirect_to DrupalNode.find(@node.power_tag('redirect')).path, notice: "You have been redirected to this page randomly for testing purposes. We apologize for any inconvenience caused."
```

Comment:
Ah, well, I don't think we need to tell people who are not mods/admins -- that would affect the test we're trying to run. But I do like the syntax, thanks! 

Could we remove the notice here, and then we're good to go? Thanks, and especially thanks for your helping people out while I was traveling!